src/components/settings/Settings.h

```diff
@@ -9,7 +9,7 @@ namespace Pinetime {
     class Settings {
     public:
       enum class ClockType : uint8_t { H24, H12 };
-      enum class Notification : uint8_t { ON, OFF };
+      enum class Notification : uint8_t { On, Off, Sleep };
```

Comment:
I understand why you changed `ON` to `On`, as `SLEEP` somehow feels out of place. But for me `On` and `Off` in code as keywords/enums feels out of place as well. Would it be ok to keep `ON` and `OFF` and just add `Sleep` as new entry?

```suggestion
      enum class Notification : uint8_t { ON, OFF, Sleep };
```

Reply:
All uppercase seems more out of place to me, since we don't use that style anywhere else.

Reply:
I agree with @Riksu9000, `Notification::On` and `Notification::Off` are more consistent with the naming convention used in the project.

Reply:
ok :)